tests/testIndexReference.py

```diff
@@ -0,0 +1,168 @@
+from __future__ import absolute_import, division, print_function
```

Comment:
I think it is important to have tests of the tasks themselves. I'll file a ticket to add a test of the file loader to that file. You do have a test that tests the indexer, and I am making the assumption that if the task behaves, then it will behave when used by the indexer.